### DISCUSSION TITLE
There is no "integration" page here! Where is the "user id"?

### DISCUSSION THREAD

**Discussion by xr0master** [emailjs-com/help#1] [Q&A] [answered]
I was trying to use your service, but when I tried to integrate it into my code, I realized that I can't see the Integration tab, is this normal? How do I fix it?

[spelling preserved]

**Comment by xr0master** [comment, marked as answer]
The integration section has been **removed**. The content was moved to the [Account page](https://dashboard.emailjs.com/admin/account).

You will find your **Public** and **Private** keys there. The Public key replaces the User ID.

**Reply by caroleatierney** [reply]
This is still being generated though:

      (function () {
          emailjs.init("<YOUR USER ID>"); // Obtain your user ID at the dashboard https://dashboard.emailjs.com/integration
      })();

**Reply by xr0master** [reply]
Sorry, I am not sure I got you.

**Reply by caroleatierney** [reply]
Sorry - this must have been autogenerated and I couldn't find integration so then I found this information on github. Maybe the autogeneration should be revised

  (function () {
      emailjs.init("<YOUR PUBLIC KEY>"); // Obtain your Public Key from your account page at the dashboard https://dashboard.emailjs.com/admin/account/general
  })();
